config/locales/chapters.yml

```diff
@@ -0,0 +1,85 @@
+en:
+  pages: 
+    chapters: 
+      headline: Start a Blueprint Chapter
+      subtitle:
+        Blueprint was founded at UC Berkeley in Spring 2013. 
+        Since then we started scaling our impact and further mission by establishing chapters in different colleges around the world. 
+      chapters:
+        - name: University of Waterloo  
+          description: UWaterloo is our first Blueprint chapter, founded in 2015 🍁
+          url: "https://uwblueprint.org/"
+          image: "chapters/waterloo.png"
+        - name: UCLA
+          description: Blueprint at the University of California, Los Angeles was established in 2019 🚀
+          url: "https://lablueprint.org/"
+          image: "chapters/ucla.png"
+        - name: University of Toronto 
+          description: We were very excited to establish the second Canadian chapter in 2020 🌿   
+          url: "http://uoftblueprint.org/"
+          image: "chapters/toronto.png"
+        - name: UC Irvine
+          description: Our second Southern California Blueprint chapter was established in 2020 ♥️
+          url: "https://uciblueprint.org/#/"
+          image: "chapters/uci.png"
+        - name: Carleton University 
+          description: Our third Blueprint chapter in Canada 🍂
+          url: "https://cublueprint.org/"
+          image: "chapters/carleton.png"
+        - name: University of Alabama
+          description: We were very proud to bring yet another BP chapter in a non-tech spot ✨
+          url: "https://website.sdwight.workers.dev/"
+          image: "chapters/alabama.png"
+        - name: University of Colorado Boulder
+          description: We are currently in the process of establishing in CU Boulder 💕
+          url: ""
+          image: "chapters/colorado.png"
+
+      why-blueprint:
+        title: Why Blueprint? 
+        description: >
+
+        bridge: Bridge the gap between tech and non-profits. 
+        bridge_description: >
+          It’s true that non-profits lack technical resources to do their meaningful work. It’s also true that tech can’t solve global challenges. 
+          Therefore, we are creating collaborative spaces both for NPOs and aspiring technologists to make positive changes together! 
+        mark: Make a mark in your local community. 
+        mark_description: > 
+          Whether your university has a lot of clubs or not, you can be the one to bring accessible pathways to get into tech for a range of diverse voices. 
+          We want to further our DEI efforts with you! 
```

Comment:
This might be me overanalyzing but this kind of reads like "We want to use you to further our DEI efforts!" -- I wonder if there is a way to change that a bit?

Reply:
Yeah, sorry! I didn't overlook this because was just copying and pasting from figma